spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LSTMPeephole.scala

```diff
@@ -76,11 +76,9 @@ class LSTMPeephole[T : ClassTag] (
         .add(SelectTable(1))
         .add(NarrowTable(2, 2)))
 
-  override def preTopology: AbstractModule[Activity, Activity, T] =
-    Sequential()
-    .add(Dropout(p))
-    .add(TimeDistributed(Linear(inputSize, hiddenSize * 4, wRegularizer = wRegularizer,
-      bRegularizer = bRegularizer)))
+  override var preTopology: TensorModule[T] =
+    Linear(inputSize, hiddenSize * 4, wRegularizer = wRegularizer,
```

Comment:
I think this is a bug - we should turn off preTopology if p is non zero, just like other cells.